examples/classification/convert_from_original/convert_vgg.py

```diff
@@ -0,0 +1,37 @@
+import chainer
```

Comment:
Considering consistency with other examples, this file should be put under `examples/vgg/`.
And it is better to convert from `.caffemodel`.